modules/ROOT/pages/_partials/sync/data-sync-technical-overview.adoc

```diff
@@ -0,0 +1,27 @@
+[id="data-sync-technical-overview-{context}"]
+= Data Sync Technical Overview
+
+This section describes the technical aspects of Data Sync.
+
+// tag::excludeDownstream[]
+
+image::data-sync-technical-overview.png[Data Sync Technical Overview]
```

Comment:
@jstaffor  do we have a source for this?

Reply:
You can find the image here @finp : https://docs.google.com/presentation/d/1CUd1Gcx3mGKVjKAURVJ49so0NANG0snqwIJU-YPYKAM/edit#slide=id.g50c800615d_7_169

Reply:
We do. The main source for all the image work pertaining to this PR can be seen here: https://issues.jboss.org/browse/AGMS-315

It was provided to us by Engineering. If you need anything else, let me know :+1: